src/cli/plugin/setting_parser.js

```diff
@@ -22,6 +22,16 @@ export default function createSettingParser(options) {
     return 'https://download.elastic.co/' + settings.organization + '/' + settings.package + '/' + filename;
   }
 
+  function areMultipleOptionsChosen(options, choices) {
+    let numChosen = 0;
+    choices.forEach(function (choice) {
+      if (options[choice]) {
+        ++numChosen;
+      }
+    });
+    return (numChosen > 1);
```

Comment:
I'd recommend doing this by checking the length of the intersection rather than imperatively counting the duplicates.  You can import `intersection` from the `lodash` module, and then this function becomes:

```js
function areMultipleOptionsChosen(options, choices) {
  return intersection(Object.keys(options), choices).length > 1;
}
```